pkg/apis/kops/networking.go

```diff
@@ -155,7 +155,7 @@ type AmazonVPCNetworkingSpec struct {
 	ImageName string `json:"imageName,omitempty"`
 }
 
-const CiliumDefaultVersion = "v1.0-stable"
```

Comment:
Are we really upgrading from 1.0 to 1.6.1?

Reply:
What’s more concerning to me: did we really not care about chili um for 6 minor versions?

Reply:
Anyone is welcome to update at anytime, but especially when leaping versions we need to ensure a smooth upgrade path for all users.